book/src/configuration.md

```diff
@@ -11,8 +11,37 @@ This example file demonstrates the default configuration:
 [general]
 # Directories to search for templates, relative to the crate root.
 dirs = ["templates"]
+# Unless you add a `-` in a block, whitespace won't be trimmed.
+automatic_trim_block = false
 ```
 
+About `automatic_trim_block`, if you want to trim whitespace before or after a
+block, you need to use the minus sign. For example:
+
+```jinja
+<div>
+
+
+{%- if something %}
+Hello
+(% endif %}
+```
+
+In this case, only the content between `<div>` and `{%-` will be trimmed. If
+this option is enabled, all content before and after each jinja block will be
+trimmed.
+
+If the option is enabled but you don't want a part to be trimmed, use the plus
+(`+`) sign:
+
+```jinja
+{% if something +%}
+Hello
+(%+ endif %}
+```
+
+In this example, `Hello` will be surrounded with backlines.
```

Comment:
I'm not familiar with "backlines", use "newline characters" instead?

Reply:
:+1: